### ISSUE TITLE
iOS版开发上有技术上的难点吗？

### ISSUE THREAD

**Issue by Siceberg** [getlantern/forum#22]
iOS版至今没有什么苗头，是否有技术上不可达到的阻碍。最近opera出来iOS的VPN，但是没几天就不能用了。

**Comment by percyalpha** [comment]
目前iOS版还没有正式开始开发。

**Comment by ChineseMao** [comment]
祈求快点

**Comment by Siceberg** [comment]
哪怕先做个免费的ss客户端也OK

**Comment by aeroxy** [comment]
I don't think so, although of course there are obstacles. Ubuntu version of Lantern is a deb pack, which is practically the same deb pack used for jailbreak tweaks but with different UI and application interfaces. Anyone with jailbreak tweaks experience joining the lantern dev community will result at a usable version of lantern at least outside of China (considering China bot net has an impressive speed to shut down lantern connections).

**Comment by huiyu998** [comment]
祈求快点
